logistics_project/apps/malawi/templates/malawi/new/reports/dashboard.html

```diff
@@ -3,7 +3,7 @@
 {% block report_content %}
 
 <form method="get" id="selector">
-	{% include "malawi/partials/location_selector.html" %}
+	{% include "malawi/partials/location_selector.html" with exclude_facilities=base_level_is_facility %}
     <input type="submit" value="Go!" />
 </form>
 
```

Comment:
(I had rolled my local clock back to 9/30 to see the September reports and then accidentally made this commit so its date is wrong. It doesn't seem like it will cause any issues besides a wrong displaying order on github so doesn't seem worth the trouble to fix)